storey/drivers.py

```diff
@@ -293,20 +312,30 @@ def _build_conditioned_feature_store_request(self, aggregation_element, pending=
 
     def _build_simplified_feature_store_request(self, aggregation_element):
         expressions = []
+        feature_attrs = ["a", "b"]
 
         times_update_expressions = {}
         new_cached_times = {}
         pending_updates = {}
         initialized_attributes = {}
+
+        # build a dictionary to save operations that can be paralleled
+        if self._parallel_ops:
+            pexpressions = {key: {val: None for val in feature_attrs} for key in self.parallel_aggregates.keys()}
```

Comment:
`.keys()` is redundant.

Reply:
`key` is a bit misleading here. Better call it `parallel_aggregate` or `aggregate` or `aggr`.